tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/model/NewLiferayPluginProjectOp.java

```diff
@@ -17,28 +17,28 @@
 import com.liferay.ide.core.ILiferayProjectProvider;
```

Comment:
I'm fine for reusing and I'm fine for modifying the original ones to make
them easier to reuse, but I just don't want to rename the originals.
 thanks.

On Sun, Nov 15, 2015 at 11:35 PM, simon.jiang <notifications@github.com>
wrote:

> In
> tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/model/NewLiferayPluginProjectOp.java
> <https://github.com/gamerson/liferay-ide/pull/1233#discussion_r44890252>:
>
> > @@ -17,28 +17,28 @@
> >  import com.liferay.ide.core.ILiferayProjectProvider;
>
> I renamed them just want to easily distinguish the class with name and I
> also want to reuse these codes what we have wrote service classes, then we
> just need to add a few different implementation of plugin project and
> module project and we don't need to maintain two copies of similar fucntion
> classes
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gamerson/liferay-ide/pull/1233/files#r44890252>.
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com